src/lib/krb5/rcache/ser_rc.c

```diff
@@ -64,15 +64,15 @@ krb5_rcache_size(krb5_context kcontext, krb5_pointer arg, size_t *sizep)
     size_t              required;
 
     kret = EINVAL;
-    if ((rcache = (krb5_rcache) arg)) {
+    if ((rcache = (krb5_rcache)arg) && (rcache->ops != NULL)) {
```

Comment:
Further down krb5_rc_get_name is passed rcache as an argument and dereferences rcache->ops in its body